src/Raven.Server/Documents/Indexes/Debugging/IndexDebugExtensions.cs

```diff
@@ -426,7 +431,10 @@ private static BlittableJsonReaderObject GetReduceResult(ulong reduceKeyHash, In
                      .Query(query, null, fieldsToFetch, new Reference<int>(), new Reference<int>(), retriever, ctx, null, CancellationToken.None)
                     .ToList();
 
-                if (result.Count != 1)
+                if (result.Count == 0)
+                    return context.ReadObject(new DynamicJsonValue(), "debug-reduce-result");
+
+                if (result.Count > 1)
```

Comment:
But that is what happened here, no? We had two values for the same reduce key

Reply:
No, what happened here was the case above - `result.Count == 0` because we filtered out the results in a given page during the reduce phase.